utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -359,17 +361,17 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
 
     int resize_w = width, resize_h = height; // height and width
 
-    RocalImage image0 = input1;
-    // RocalImage image0 = rocalResize(handle, input1, resize_w, resize_h, false); // uncomment when processing images of different size
-    RocalImage image1;
+    RocalTensor image0 = input;
+    // RocalTensor image0 = rocalResize(handle, input, resize_w, resize_h, false); // uncomment when processing images of different size
```

Comment:
Since we removed resize
Does the display of outputs works as expected? I remember this resize is done to avoid the overlapping of outputs of different sizes
Please check if applicable

Reply:
Each output will be dumped as separate image so resize will not be necessary